src/dotnet/APIView/APIViewWeb/README.md

```diff
@@ -0,0 +1,80 @@
+
+### C
+1. Install clang 10 or later.
+2. Run clang [inputs like az_*.h] -Xclang -ast-dump=json -I ..\..\..\core\core\inc -I "c:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.26.28801\include\" > az_core.ast
+3. Archive the file Compress-Archive az_core.ast -DestinationPath az_core.zip
+4. Upload the resulting archive using `Create Review` link in APIView.
+
+### C++
+1. Install clang 10 or later. Your environment may require additional include directories.
+2. Run clang++ [input like .\sdk\storage\azure-storage-files-datalake\inc\azure\storage\files\datalake\datalake.hpp] -I .\sdk\storage\azure-storage-files-datalake\inc -I .\sdk\core\azure-core\inc -Xclang -ast-dump -I .\sdk\storage\azure-storage-common\inc -I .\sdk\storage\azure-storage-blobs\inc > Azure_Storage_Files_Datalake.ast
+3. Archive the file Compress-Archive Azure_Storage_Files_Datalake.ast -DestinationPath Azure_Storage_Files_Datalake.zip
+4. Rename the file Rename-Item Azure_Storage_Files_Datalake.zip -NewName Azure_Storage_Files_Datalake.cppast
+5. Upload the resulting archive using `Create Review` link in APIView.
+
+### Go
+1. Archive source module directory in which go.mod is present. Compress-Archive ./sdk/azcore -DestinationPath azcore.zip
+2. Rename the file Rename-Item azcore.zip -NewName azcore.gosource
+3. Upload the resulting archive.
+
+### Java
+1. Run `mvn package` build on your project, which will generate a number of build artifacts in the `/target` directory. In there, find the file ending `sources.jar`, and select it.
+2. Upload the selected file to APIView tool using `Create Review` link.
+
+### JS/TS
+1. Use api-extractor to generate a [docModel](https://api-extractor.com/pages/setup/generating_docs/) file
+2. Upload generated api.json file using `Create Review` link.
+
+### Python
+1. Generate wheel for the package. python setup.py bdist_wheel -d [dest_folder]
+2. Upload generated whl file
+
+
+## How does it retrieve public API information
+
+Developers can upload package or an abstract file generated based on each language into API View tool. APIView tool has a language processor for each language it supports and these individual language processor extracts API stub information and create json APIView file that is processed by tool. API review tool stores original uploaded file and generated json file in its data store. Review pages are rendered using this generated json file which contains tokens to present language keywords, links etc. Tool also allows user to recreate review json from stored original file if language package processor itself is updated.
+
+## Types of API reviews
+
+APIView tool shows three different types of reviews based on how it is generated.
+- Manual
+- Automatic
+- Pull request reviews
+
+### Manual
+Manual reviews are created by developers vy uploading artifact as per teh instructions given above. This will allow developers to review API changes if API review is not available from PR branch.

```

Comment:
```suggestion
Manual reviews are created by developers by uploading artifact as per the instructions given above. This will allow developers to review API changes if API review is not available from PR branch.

```